### PR TITLE
Return referrals at /activities

### PR DESCRIPTION
**Pull request by murilovmachado**
Jira: https://bugs.benetech.org/browse/CHI-375

This PR adds support for referrals at the `GET /cases/{caseId}/activities` endpoint.

### CODE REVIEW

File: controllers/activities.js [hunk 1]
```diff
@@ -14,9 +15,13 @@ const ActivityTypes = {
 
 const isConnectContactType = type => Object.keys(ActivityTypes.connectContact).includes(type);
 
+function getObjectsFromInfo(caseObj, name) {
```
Comment (nickhurlburt) [inline controllers/activities.js:18]: Good addition

File: controllers/activities.js [hunk 2]
```diff
@@ -28,6 +33,23 @@ function createAddNoteActivity({ previousValue, newValue, createdAt, twilioWorke
   };
 }
 
+function createAddReferralActivity({ previousValue, newValue, createdAt, twilioWorkerId }) {
```
Comment (nickhurlburt) [inline controllers/activities.js:36]: We've created an O(N^2) algorithm to populate an activity list.  For now we can live with it, but it seems rather unnecessary.  Could you create a jira ticket for future debt work that lists some ideas for how we could improve this?

**Review by nickhurlburt — APPROVED** [review]
Looks good

File: controllers/activities.js (hunk 2 above)
Comment (murilovmachado) [reply controllers/activities.js:36]: https://bugs.benetech.org/browse/CHI-436

**Pull request merged**